test/features/feed.js

```diff
@@ -41,6 +41,27 @@ describe('Featured feed', () => {
         .finally(() => assert.cleanupRecorder());
     });
 
+    it('Should forward accept-language', () => {
+        const date = '2018/05/07';
+        assert.recordRequests();
+        return preq.get({
+            /* zh.wikipedia.beta.wmflabs.org doesn't have data for /feed/featured tests */
+            uri: `${server.config.baseURL('zh.wikipedia.org')}/feed/featured/${date}`,
```

Comment:
@Pchelolo zh.wikipedia.org is momentarily being used unless it's not possible to create actual data to be tested in the beta cluster.

Reply:
I guess pageviews is [not enabled](https://deployment.wikimedia.beta.wmflabs.org/api/rest_v1/#/) for the beta cluster and wikifeeds is using [pageviews](https://wikimedia.org/api/rest_v1/#/Pageviews%20data) to create the mostread feed. Is there a problem to keep using `zh.wikipedia.org` for this new test?

Reply:
As I said, having prod wikis in tests is problematic since all the backend services we're talking to are in beta and technically should not have access to prod wikis.

Reply:
You can though mock response from wikifeeds, or indeed create some pages in beta.

Reply:
Creating pages in beta didn't help because the pageviews service is only available for production. I dropped the integration tests here since wikifeeds already handle unit tests.